### DISCUSSION TITLE
<Subscribe> not working

### DISCUSSION THREAD

**Discussion by afroguy16** [re-rxjs/react-rxjs#264] [General]
Hi, I tried to implement this with my knowledge of how it works in Angular, that is, handle the subscription by myself. It worked right, but it was failing in RTL. So I followed the documentation and used the `<Subscribe></Subscribe>` component to wrap my component, but I was getting an error `Uncaught [Error: Missing Subscribe!]`. So I brought down my implementation and followed the docs word for word, but the error didn't go away. Below is the code and the error: 
```import { bind, Subscribe } from "@react-rxjs/core"
import { createSignal } from "@react-rxjs/utils"
import { map } from "rxjs/operators"

// A signal is an entry point to react-rxjs. It's equivalent to using a subject
const [textChange$, setText] = createSignal<string>();

const [useText, text$] = bind(textChange$, "")

function TextInput() {
  const text = useText()

  return (
    <div>
      <input
        type="text"
        value={text}
        placeholder="Type something..."
        onChange={(e) => setText(e.target.value)}
      />
      <br />
      Echo: {text}
    </div>
  )
}

// Previously...
// const [useText, text$] = bind(...);

const [useCharCount, charCount$] = bind(
  text$.pipe(
    map((text) => text.length)
  )
)

function CharacterCount() {
  const count = useCharCount()

  return <>Character Count: {count}</>
}

function CharacterCounter() {
    return (
      <div>
        <Subscribe>
          <TextInput />
          <CharacterCount />
        </Subscribe>
      </div>
    )
  }

export default CharacterCount
```

![image](https://user-images.githubusercontent.com/15367904/166484055-c1e07f8f-d565-41ae-b323-950e8d8ad407.png)

##Steps to Produce
1. Create react app with typescript template
2. Copy paste the code from the [documentation](https://react-rxjs.org/docs/getting-started)
3. Run `npm start`

**Comment by josepot** [comment]
Hi @afroguy16 ! Thanks for reporting this. Unfortunately, though, [I can't manage to reproduce the issue](https://codesandbox.io/s/flamboyant-dijkstra-bkgycv?file=/src/App.tsx).

Could you please provide us with a [codesandbox](https://codesandbox.io/), [stackblitz](https://stackblitz.com/) or a github repo that we can use to reproduce this issue?

Thanks!

**Reply by afroguy16** [reply]
Hi @josepot, thanks for trying. I also tried to reproduce it in a new project, but it works fine. I will try to debug further and drop any findings I see.
